src/diffusers/loaders/single_file.py

```diff
@@ -181,6 +181,31 @@ def from_single_file(cls, pretrained_model_link_or_path, **kwargs):
             revision (`str`, *optional*, defaults to `"main"`):
                 The specific model version to use. It can be a branch name, a tag name, a commit id, or any identifier
                 allowed by Git.
+            original_config_file (`str`, *optional*):
+                The path to the original config file that was used to train the model. If not provided, the config file
+                will be inferred from the checkpoint file.
+            model_type (`str`, *optional*):
+                The type of model to load. If not provided, the model type will be inferred from the checkpoint file.
+            image_size (`int`, *optional*):
+                The size of the image output. It's used to configure the `simple_size` parameter of the UNet and VAE model.
+            load_safety_checker (`bool`, *optional*, defaults to `False`):
+                Whether to load the safety checker model or not. By default, the safety checker is not loaded unless user pass
+                a `safety_checker` component in the `kwargs`.
```

Comment:
Should we maybe include a note that this doesn't apply to models like SDXL or even better something like "This doesn't apply to all models. Such as: SDXL." WDYT?